packages/config/src/forkConfig/index.ts

```diff
@@ -117,7 +117,7 @@ export function createForkConfig(config: ChainConfig): ForkConfig {
 
   // Convert basis points to milliseconds into the slot
   const getSlotComponentDurationMs = (basisPoints: number): number =>
-    Math.floor((basisPoints * config.SLOT_DURATION_MS) / BASIS_POINTS);
+    Math.round((basisPoints * config.SLOT_DURATION_MS) / BASIS_POINTS);
```

Comment:
I think this would cleaner, noticed attestation deadline is 3999ms otherwise, not as per spec but doesn't matter in this case

Reply:
Yea worst case is +/- 0.5ms, which is negligible